### ISSUE TITLE
No using active schema for any SQL file with Oracle

### ISSUE THREAD

**Issue by mkgrgis** [dbeaver/dbeaver#8766]
[x:oracle]
#### System information: 
- Operating system (distribution) and version
Linux Mint, Ubuntu 20.04, RHEL 8
- DBeaver version
7.0.4
- Additional extensions
No
#### Connection specification:
- Database name and version
Oracle 10.0
- Driver name
from maver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
No

#### Describe the problem you're observing:
1. Connect to Oracle DB
2. Ensure, there is active schema
3. Open any SQL file (for example from recent)
-> no active schema for SQL file, error if trying to execute any SQL command.

**Comment by uslss** [comment]
hi @mkgrgis ,
is it possible to set active schema in toolbar? Is it reproducible for newly created script? Is there something in logs?
thanks in advance

**Comment by mkgrgis** [comment]
is it possible to set active schema in toolbar? 
no, the same problem with connection and transaction
![Uploading 5.png…]()

Is it reproducible for newly created script?
No

UTF-8 filename (?)
![4](https://user-images.githubusercontent.com/41448637/82573243-33816200-9b8e-11ea-99b5-1c3a448ddbfa.png)

**Comment by mkgrgis** [comment]
There is only debeaver-debug.log in logs, no ".log"

**Comment by uslss** [comment]
connection is valid and works ok?

**Comment by mkgrgis** [comment]
yes, but for activities with the file there is window "connection not valid"

**Comment by mkgrgis** [comment]
If I open a SQL file and try to select db connection -> Can't open editor connection https://user-images.githubusercontent.com/41448637/82573243-33816200-9b8e-11ea-99b5-1c3a448ddbfa.png
If I try to select a schema, there is list of schemas, but no fixation on double click/enter/selection

**Comment by mkgrgis** [comment]
Today I get .log

```
!SESSION 2020-05-22 09:13:07.770 -----------------------------------------------
eclipse.buildId=unknown
java.version=11.0.7
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=ru_RU
Command-line arguments:  -os linux -ws gtk -arch x86_64

!ENTRY org.eclipse.ui 4 0 2020-05-22 09:31:55.610
!MESSAGE Unhandled event loop exception
!STACK 0
java.lang.NullPointerException
	at org.eclipse.ui.operations.OperationHistoryActionHandler.dispose(OperationHistoryActionHandler.java:221)
	at org.eclipse.ui.texteditor.AbstractTextEditor.registerUndoRedoAction(AbstractTextEditor.java:5655)
	at org.eclipse.ui.texteditor.AbstractTextEditor.dispose(AbstractTextEditor.java:4421)
	at org.eclipse.ui.texteditor.AbstractDecoratedTextEditor.dispose(AbstractDecoratedTextEditor.java:391)
	at org.jkiss.dbeaver.ui.editors.text.BaseTextEditor.dispose(BaseTextEditor.java:104)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditorBase.dispose(SQLEditorBase.java:484)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4930)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:4451)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1160)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1049)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:658)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:557)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:154)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:233)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:401)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:657)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:594)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1447)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1420)
```

**Comment by uslss** [comment]
@mkgrgis , still cannot reproduce.
Is it still reproducible for you against the recent version?
Thanks in advance

**Comment by mkgrgis** [comment]
In 7.1.1 there is no same bug with `java.lang.NullPointerException` or `connection not valid`, but there is similar bug. For example I have Oracle 11 and Postgres 12 connections with active schema and some tables.
Way to errors.
1. Start Dbeaver
2. A table from Oracle is active
![1](https://user-images.githubusercontent.com/41448637/85878759-e4100080-b7e1-11ea-8f04-b1bbeda66ec8.png)
3. Drag-drop any SQL script for Oracle to panel at right of table title
![2](https://user-images.githubusercontent.com/41448637/85879362-e4f56200-b7e2-11ea-8824-1bf9d99c6adb.png)
=> error!
4. Select active schema in Postgres
5. Drag-drop other SQL script for Postgres
![3](https://user-images.githubusercontent.com/41448637/85879461-14a46a00-b7e3-11ea-9b94-3b16e4bd4a46.png)
=> active connection is Oracle, error!

**Comment by mkgrgis** [comment]
A similar issue https://github.com/dbeaver/dbeaver/issues/9040

**Comment by uslss** [comment]
closed as duplicate of #9040
